interface/client/templates/views/tab.js

```diff
@@ -50,14 +49,24 @@ Template['views_tab'].onRendered(function(){
         // Meteor.clearTimeout(timeoutId);
         TemplateVar.set(template, 'loading', false);
 
+        var titleFull = webview.getTitle(),
+            title;
+
+        if(titleFull && titleFull.length > 40) {
```

Comment:
The point is we don't want to add the tex overflow on any element where we use that. 
Titles more than 40 characters are uncommon and for that i added the property "nameFull"